src/client/routes/Market/index.tsx

```diff
@@ -72,12 +82,15 @@ export const Market = () => {
 
     const expectedCategories = _.keys(defaultLineCategories);
     const currentCategories = _.keys(lineCategoriesForDisplay);
-
+    console.log(expectedCategories);
     // const shouldFetch = expectedCategories.reduce((bool, cat) => bool && cuirrentCategories.includes(cat), true);
     let shouldFetch: boolean = false;
     expectedCategories.forEach((cat) => (shouldFetch = shouldFetch || !currentCategories.includes(cat)));
 
-    if (shouldFetch) fetchMarketData();
+    if (shouldFetch) {
+      fetchMarketData();
+      console.log();
```

Comment:
rm

Reply:
the whole block? Or just the console log?

Reply:
fixed